node/pkg/governor/governor.go

```diff
@@ -606,12 +750,17 @@ func (gov *ChainGovernor) CheckPendingForTime(now time.Time) ([]*common.MessageP
 							Hash:           pe.hash,
 						}
 
-						if err := gov.db.StoreTransfer(&xfer); err != nil {
+						if err := gov.db.StoreTransfer(&dbTransfer); err != nil {
 							gov.msgsToPublish = msgsToPublish
 							return nil, err
 						}
 
-						ce.transfers = append(ce.transfers, &xfer)
+						transfer, err := newTransferFromDbTransfer(&dbTransfer)
+						if err != nil {
+							return nil, err
+						}
+						ce.transfers = append(ce.transfers, transfer)
+						// TODO: Add inverse transfer to destination?
```

Comment:
Yeah I think if this is a transfer that fits in the daily limit (i.e. counts towards transfers) then it should have the inverse on the destination to flow cancel as well...wdyt?

Reply:
@bruce-riley do you have any thoughts here? I'm not exactly sure how pending transfers should be handled

Reply:
I think that makes sense.